mkdocs.yml

```diff
@@ -93,5 +93,4 @@ nav:
         - LifecycleManager: developers/lifecyclemanager.md
         - ShuffleClient: developers/shuffleclient.md
       - Fault Tolerant: developers/faulttolerant.md
-#      - ReadData: developers/readdata.md
-#      - Slots Allocation: developers/slotsallocation.md
+      - Blacklist: developers/blacklist.md
```

Comment:
can still use blacklist concept as we already rename blacklist to excluded

Reply:
should be "worker exclusion"